hooks/check-commits.sh

```diff
@@ -26,8 +27,11 @@ trap finish EXIT
 hashes=$(git rev-list "$1")
 for h in ${hashes}
 do
+	echo Checking $h... >&2
```

Comment:
Is there an option to only print this when there is an error?
All the check scripts stay silent when everything is ok.

Reply:
Maybe check-diff.py or check-message.py could output it themselves?

Reply:
I guess easiest is to pipe all output to another temporary file, and echo that to stderr at the end, if failure was detected.

Reply:
Seems marginally nasty. How about just outputting stuff on error, and checking for empty string?

Reply:
For me always displaying the commit being checked when there's no error is ok. And it would be even better to also display the commit message and not only the hash. Easier for contributors to identify where the error is.

Reply:
Make also prints the names of files being compiled as progress indicators. But the compiler still prefixes each message with the name of the file causing the message.